### DISCUSSION TITLE
Docker Image is out of Date

### DISCUSSION THREAD

**Discussion by Ryonez** [dani-garcia/vaultwarden#1528] [Q&A]
The code for send was added 6 days ago, however the docker images haven't been updated yet. There's only been some testing images made, has the image automation possibly died?

Could this be looked into please?

**Comment by cloverzrg** [comment]
Not released yet

**Comment by Ryonez** [comment]
Ahh, might explain it. I only found out the code had been added as it's been added as a supported feature in the wiki.

**Comment by BlackDex** [comment]
Use the `testing` tag it is based upon the master branch.
